src/apps/material/material.tsx

```diff
@@ -28,6 +37,40 @@ const Material: React.FC<MaterialProps> = ({ pid, searchUrl }) => {
     return <div>No work data</div>;
   }
 
+  const {
+    manifestations,
+    titles,
+    materialTypes,
+    mainLanguages,
+    creators,
+    workYear
+  } = data.work;
+
+  const creatorsText = creatorsToString(
+    flattenCreators(filterCreators(creators, ["Person"])),
+    t
+  );
+
+  const listDescriptionData = {
+    Type: {
+      value: materialTypes?.[0]?.specific,
+      type: "standard"
+    },
+    Sprog: { value: mainLanguages?.[0].display, type: "standard" },
```

Comment:
Consider if we should show all values for language and types.

As far as I can tell this will only show the first value is shown but our schema indicates that there may be more.